analysed-packages/dbus/version-1.14.6/dbus-1.14.6-SPDX2TV.spdx

```diff
@@ -0,0 +1,13415 @@
+(or, at your option any later version).
+
+Thus we concluded  GPL-2.0-or-later OR AFL-2.1 as license of this file. </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-AFL-2.1
+LicenseInfoInFile: LicenseRef-Dual-license
+FileCopyrightText: <text> Copyright (C) 2007 Red Hat, Inc. </text>
+FileAttributionText: <text> To the extent that files may be licensed under either GPL-2.0-or-later or AFL-2.1, in this context, AFL-2.1 has been chosen. This choice shall not restrict the freedom of other users to select either GPL-2.0-or-later or AFL-2.1 licenses. For convenience, all license texts are provided. </text>
+
+##File
+
+FileName: dbus-1.14.6.tar.xz/dbus-1.14.6/COPYING
+SPDXID: SPDXRef-item546376
+FileChecksum: SHA1: 5f0d0745072e82c4dcebc53223dcec8dc1aaebc3
+FileChecksum: SHA256: e61807cd1c32ff4e7bd5b4b61dd21997c6dc5642cf19316124fe38d50e1f9fa3
+FileChecksum: MD5: 6423dcd74d7be9715b0db247fd889da3
+LicenseConcluded: LicenseRef-License-of-GNU-Licenses
+LicenseComments: <text> The file contains the text of the AFL-2.1 and the text of the GPL-2.0-or-later. The text of the GPL-2.0-or-later is licensed under this license:
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+Thus we concluded this license as license of the part of the file which contains the text of the GPL-2.0-or-later. </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-AFL-2.1
+LicenseInfoInFile: LicenseRef-GPL-2.0-only
+LicenseInfoInFile: LicenseRef-Dual-license
+LicenseInfoInFile: LicenseRef-License-of-GNU-Licenses
+FileCopyrightText: <text> Copyright (C) 2003-2004 Lawrence E. Rosen. All rights reserved. 
+Copyright (C) 1989, 1991 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: dbus-1.14.6.tar.xz/dbus-1.14.6/dbus/versioninfo.rc.in
+SPDXID: SPDXRef-item546525
+FileChecksum: SHA1: 069097425213015aced2d5ed55efa13c67132cc3
+FileChecksum: SHA256: c8642bd186e385597edbd4c52689d5cf1af225fd4fdd912b74a10a8cb85f26e5
+FileChecksum: MD5: 7ce971c771bdc7bd634e8a216c502905
+LicenseConcluded: LicenseRef-FSFULLRWD-b58551cc823ae75cade713563ea7735d
+LicenseInfoInFile: LicenseRef-AFL-2.1
+LicenseInfoInFile: LicenseRef-GPL-2.0-only
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-FSFULLRWD
+FileCopyrightText: <text> Copyright (C) 2005 g10 Code GmbH
+Copyright © 1994-2020 dbus contributors </text>
+
+##File
+
+FileName: dbus-1.14.6.tar.xz/dbus-1.14.6/aclocal.m4
+SPDXID: SPDXRef-item546278
+FileChecksum: SHA1: f73f5c08165be544b68664d71e65a0a6b0dbbebe
+FileChecksum: SHA256: 02aec25ae64e7db4e8b5836ce3c51fc4c15bf8a1fd32235932e2b5482f8637f9
+FileChecksum: MD5: 646367a548dbafe053196ed08f09c5f5
+LicenseConcluded: LicenseRef-FSFAP AND LicenseRef-FSFAP-no-warranty-disclaimer AND LicenseRef-FSFULLR AND LicenseRef-FSFULLRWD-1ac11883dcc7dd1681da7708ef7e33fd AND LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
```

Comment:
Thank you very much for thorough review!
I picked FSFAP-no-warranty-disclaimer because  if you look at monk#13, it does not include "**This file is offered as-is, without any warranty.**" at the end of the license text and that is the main difference between LicenseRef-FSFAP and  LicenseRef-FSFAP-no-warranty-disclaimer.  LicenseRef-FSFAP-no-warranty-disclaimer does not  include this sentence.
![ LicenseRef-FSFAP-no-warranty-disclaimer](https://github.com/Open-Source-Compliance/package-analysis/assets/130649329/06c8103c-2471-410b-bd52-444864f34941)

![LicenseRef-FSFAP](https://github.com/Open-Source-Compliance/package-analysis/assets/130649329/063a2eb8-4a27-4693-86e1-e6b5b5482f77)

Reply:
@ParianGol good catch, you are absolutely right. Your conclusion is correct, great

Reply:
@OliverFendt Thank you very much!